configs/linux/postinst.sh

```diff
@@ -7,7 +7,27 @@ if ! grep -q '^user_allow_other$' /etc/fuse.conf; then
 fi
 
 # Pick up changes to /etc/logind.conf.
-systemctl restart systemd-logind.service
+if [ -z "$(loginctl --no-legend list-sessions)" ]; then
+  systemctl restart systemd-logind.service
+else
+  # Restarting systemd-logind.service effectively kills user sessions and the
+  # users' desktop environment, so let the user get a hint about reboot
+  # instead.
+  if [ "$#" -ge 2 ] && [ "$1" = "configure" ]; then
```

Comment:
Nit: can we be a bit consistent here with how we call `[` and/or `test`?

On this line we can use `[`'s own `-a` operator:

```sh
if [ "$#" -ge 2 -a "$1" = "configure" ]; then
```

And a couple of lines below we're using `test` instead of `[`:

```
if [ -e /var/run/reboot-required ]; then
```

Reply:
I went for calling `[`. I see that you use `[` in some scripts and `test` in others.